### ISSUE TITLE
Can’t send text mail with embed categories.

### ISSUE THREAD

**Issue by retdaisuke** [s-ichikawa/laravel-sendgrid-driver#91]
Hi, I have a problem unable to send text mail with embed categories.

Laravel 5.5.45
s-ichikawa/laravel-sendgrid-driver 2.0.5

This code was sending a mail as "text/html" , not "text/plain"

```php
Mail::send(['text' => 'emails.order_received'], $data, function ($message) use ($data) {
    $message->subject('Order received');
    $message->from('foo@example.com');
    $message->to('bar@example.com');

    $message->embedData([
        'categories' => ['Order'],
    ], 'sendgrid/x-smtpapi');
});
```

Here is a work-around. Set a additional parameter to set content type "text/plain".

```php
Mail::send(['text' => 'emails.order_received'], $data, function ($message) use ($data) {
    $message->subject('Order received');
    $message->from('foo@example.com');
    $message->to('bar@example.com');

    $message->embedData([
        // In the case of using categories, set content type "text/plain" to avoid to be "text/html"
        'content' => [
            ['type' => 'text/plain', 'value' => $message->getBody()]
        ],
        'categories' => ['Order'],
    ], 'sendgrid/x-smtpapi');
});
```

Apparently with embed data (categories etc.), `$message->getContentType()`  is  seem to be  `multipart/related`. 
 [https://github.com/s-ichikawa/laravel-sendgrid-driver/blob/master/src/Transport/SendgridTransport.php#L162](https://github.com/s-ichikawa/laravel-sendgrid-driver/blob/master/src/Transport/SendgridTransport.php#L162) 

This is a reason. But, it's `Swift_Mime_SimpleMessage` behavior. 
Sorry, I don't have a good pull request...

Cf. This is doesn't matter
 [https://github.com/s-ichikawa/laravel-sendgrid-driver/issues/51](https://github.com/s-ichikawa/laravel-sendgrid-driver/issues/51)

**Comment by s-ichikawa** [comment]
Thank you for your issue.

This problem does not happen by recently version Laravel, right?
This reason seems that Mailer's content-type set timing was changed after this commit is merged.
https://github.com/laravel/framework/commit/0cd889942c7dfd17c25551aab681eb6d9a96182a#diff-bd694c597e7855f76d9f110495ece691

Recently version Laravel set email body and content-type after running the function of third arguments, so content-type is decided by a template type.
But in the old version, "embedData" function overwrites content-type that decided by a template type.

I don't have a good idea now too.
I think you can use simply `$message->setContentType('text/plain');` for avoid this problem. (But this is not the smart way for this library..)
like a following:
```
Mail::send(['text' => 'emails.order_received'], $data, function ($message) use ($data) {
    $message->subject('Order received');
    $message->from('foo@example.com');
    $message->to('bar@example.com');

    $message->embedData([
        'categories' => ['Order'],
    ], 'sendgrid/x-smtpapi');

    $message->setContentType('text/plain');
});
```

**Comment by retdaisuke** [comment]
I’m sorry to slowly reply.

I checked this problem dose not happen with Laravel 5.8.3(laravle/framework 5.8.10). I want this commit to be merged to Laravel 5.5, but it’s  seem to be breaking change, so it does not come true.

Yes, I'll use `$message->setContentType('text/plain');`. It's relatively short.

Thank you!
